### PR TITLE
improving ToC and adding documentation section

### PR DESCRIPTION
**Pull request by Tovli**
Following #92 I've added a link to our official docs
While at it, I've added sub sections to our ToC

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,6 +52,9 @@ Any low or medium issues that have not been addressed have been logged in the Gi
 We welcome contributions of all kinds to Cyberark Secrets Provider for Kubernetes. For instructions on
 how to get started and descriptions of our development workflows, see our [contributing guide](CONTRIBUTING.md).
 
+# Documentation
+You can find official documentation on [our site](https://docs.conjur.org/Latest/en/Content/Integrations/Kubernetes_deployApplicationsConjur-k8s-Secrets.htm)
```
Comment (orenbm) [inline README.md:56]: please add a period at the end of the line.

**Comment by sigalsax** [comment]
@Tovli To continue your thread from slack regarding 
```
In addition, each time the master build is green, we push a <version>-<git_version> (i.e. 0.2.0-d9494c1) image to our internal repo
```

Can you also update this to be to reflect our updated workflow?; pushing to our internal repo during releases.

```
For each release, we push a <version>-<git_version> (i.e. 0.2.0-d9494c1) image to our internal repository.
```

@eladkug @orenbm correct me if you believe it should say something else

**Comment by orenbm** [comment]
> @Tovli To continue your thread from slack regarding
> 
> ```
> In addition, each time the master build is green, we push a <version>-<git_version> (i.e. 0.2.0-d9494c1) image to our internal repo
> ```
> 
> Can you also update this to be to reflect our updated workflow?; pushing to our internal repo during releases.
> 
> ```
> For each release, we push a <version>-<git_version> (i.e. 0.2.0-d9494c1) image to our internal repository.
> ```
> 
> @eladkug @orenbm correct me if you believe it should say something else

@sigalsax how is this related to the PR? can you please add it in another one?

**Comment by andresguisado** [comment]
LGTM.

**Pull request merged**